W3ChampionsStatisticService/PlayerProfiles/PlayersController.cs

```diff
@@ -61,16 +61,16 @@ public async Task<IActionResult> GetPlayer([FromRoute] string battleTag, [FromQu
                 {
                     return Unauthorized("Sorry Hackerboi");
                 }
-                
+
                 player = PlayerOverallStats.Create(battleTag);
             }
-            
+
             // Akas are stored in cache - preferences for showing akas are stored in DB
             var settings = await _personalSettingsRepository.Load(battleTag);
-            player.PlayerAkaData = _playerAkaProvider.GetAkaDataByPreferences(battleTag, settings);
+            player.PlayerAkaData = await _playerAkaProvider.GetAkaDataByPreferences(battleTag, settings);
```

Comment:
probably we should rename this `GetAkaDataByPreferencesAsync` if we're doing that elsewhere

Reply:
Done